tsg/ranker/base.py

```diff
@@ -78,19 +119,29 @@ def cosine_score_calc(query_terms, index_dictionary_path=DICTIONARY_PATH,
 
     return scores
 
+def combine_and_or_scores(and_dict, or_dict):
+    sorted_and = sorted(and_dict.items(), key = operator.itemgetter(1,0))
+    sorted_or = sorted(or_dict.items(), key = operator.itemgetter(1, 0))
+
+    new_docs = set(sorted_or) - set(sorted_and)
```

Comment:
This should not be working. you are comparing 2-tuples with weight which are always different between or_list and and_list. am I confused?